client/ayon_core/hosts/traypublisher/api/editorial.py

```diff
@@ -186,14 +186,15 @@ def _create_parents_from_settings(self, parents, data):
             # in case first parent is project then start parents from start
             if (
                 _index == 0
-                and parent_token_type == "project"
+                and parent_token_type == "Project"
```

Comment:
Where does this capitalized P come from in this `Project`?

Reply:
from settings https://github.com/ynput/ayon-core/blob/enhancement/AY-1112_Editorial-data-in-hierarchyContext/server_addon/traypublisher/server/settings/editorial_creators.py#L32

Reply:
```suggestion
                and parent_token_type.lower() == "project"
```